pennylane/devices/default_qubit_autograd.py

```diff
@@ -106,6 +106,14 @@ class DefaultQubitAutograd(DefaultQubit):
     _tensordot = staticmethod(np.tensordot)
     _conj = staticmethod(np.conj)
     _imag = staticmethod(np.imag)
+    _roll = staticmethod(np.roll)
+    _stack = staticmethod(np.stack)
+
+    def __init__(self, wires, *, shots=1000, analytic=True):
+        super().__init__(wires, shots=shots, analytic=analytic)
+        del self._apply_ops["PauliY"]
+        del self._apply_ops["Hadamard"]
+        del self._apply_ops["CZ"]

```

Comment:
😆 This is to give better benchmarks?

Reply:
Yes :sweat_smile:

Reply:
How so? :smile:

Reply:
Added: 26e9ab3b4268bae09c2abc3671f2b129ff1ae5c8